### ISSUE TITLE
UI text color dull/muted/pale (Linux)

### ISSUE THREAD

**Issue by alixyz-1** [artsyfriedchicken/EdgyArc-fr#14]
Text colour on the browser UI is dull/muted/pale. Text should be more vibrant and readable? Unsure if this is just an issue on Linux or is universal - if an intended feature, then a toggle/option would be great :)

- Linux Manjaro (KDE 5)
- Firefox 124.01
- Using main branch downloaded 24.03.24 as release V1.0.0-beta.7 has broken translucency toggle

Screenshots below: Different colours from Adaptive Tab Bar Colour show that the text is kind of hard to read and appears "washed out".
![Screenshot_20240324_195055](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/6473194/715c0c58-8314-4348-b72c-e4ba3156f619)
![Screenshot_20240324_195112](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/6473194/131cefcc-19cb-4047-aa01-e508513598ff)
![Screenshot_20240324_195019](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/6473194/ed424fe4-d5d0-4f43-9f41-bcea0e1d6963)

Screenshot: Worse when window is unfocused.
![Screenshot_20240324_200313](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/6473194/817a0542-fde7-4e3e-8295-0956d18ddc39)

**Comment by artsyfriedchicken** [comment]
couple of things 

1. `uc.tweak.af.not-translucent` was introduced in beta 7.1. please make sure you have the latest main.css and not-translucent.css
2. Unfortunately the washed out colours are an unfortunate sideeffect of trying to get translucency working in macos. the not-translucent flag should help mitigate it A BIT 
3. adaptive tab colour addon sometimes picks THE most obnoxious colours lmao. 
im trying to figure out a solution to overlay it with a light/dark colour to make it more paletable atm! 
4. yes i have the sidebar opacity go down for unfocused windows as that the default macos behaviour. if that is not something that goes with linux based window management lemme know and i can fix that in a min :)

**Comment by artsyfriedchicken** [comment]
Yeah, sure, i can put in a toggle for the dimming in my next commit later today! Just a lil something to force the toolbar and sidebar to always have 100% opacity!

The text colour on the other hand SHOULD be fixed by enabling the non-translucent flag - that literally hard sets the opacity of the sidebar to 100%, and the font colour is set to light-dark(black, white)

As for your last point, im actually working on simplifying and streamlining the code atm, hopefully itll be pretty self-explanatory once its all cleaned up and commented!!

**Comment by alixyz-1** [comment]
Hi thank you so much for that - I've just tried the latest commit and tried setting the uc.tweak.no-dimming to true and while it still dims the opacity of the text quite a bit when the window is not focused, the text on the active window is now perfectly readable and looks great!

**Comment by artsyfriedchicken** [comment]
Thats odd, it shouldnt be dimming the sidebar on inactive windows... 
Please do try the new update i just pushed? fingers crossed that solves it lmao

**Comment by alixyz-1** [comment]
Hi there, I just tried out the beta.8 release and unfortunately the issue remains. I tried installing on a fresh firefox profile and it still happens. Below is a recording of it happening in case it helps:
[Screencast_20240327_170952-2.webm](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/6473194/e5461b18-fb00-4223-9621-bba113dcb07d)
The tab sidebar opacity increases on sidebar hover and when clicked/focused the top bar opacity increases too.
Thanks for your attention to this :)
